.github/workflows/ci.yaml

```diff
@@ -198,27 +229,27 @@ jobs:
     # Step: Download feature tests app Elixir dependencies.
     # If unchanged, uses the cached version.
     - name: Download feature tests app Elixir deps
-      run: cd test/features && mix deps.get
+      run:  mix deps.get
```

Comment:
Can you remove redundant spacing after "run:  " (this happens on all the updated lines)